src/main/java/com/commercetools/sync/inventories/README.md

```diff
@@ -0,0 +1,318 @@
+# commercetools inventory sync
+
+Java module that can be used to synchronise your new commercetools inventory entries to your existing
+commercetools project.
+
+- [What it offers?](#what-it-offers)
+- [How to use it?](#how-to-use-it)
+- [How does it work?](#how-does-it-work)
+- [FAQ](#faq)
+
+
+## What it offers?
+
+1. Synchronise inventory entries coming from an external system, in any form (CSV, XML, etc..), that has been already mapped to
+[JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) 
+[InventoryEntryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntryDraft.java)
+objects to the desired commercetools project.
+2. Synchronise inventory entries coming from an already-existing commercetools project in the form of
+[JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) 
+[InventoryEntry](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntry.java)
+objects to another commercetools project.
+
+3. Build any of the following commercetools [JVM-SDK](https://github.com/commercetools/commercetools-jvm-sdk) update action
+objects given an old inventory entry, represented by a [InventoryEntry](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntry.java),
+and a new inventory entry, represented by a [InventoryEntryDraft](https://github.com/commercetools/commercetools-jvm-sdk/blob/master/commercetools-models/src/main/java/io/sphere/sdk/inventory/InventoryEntryDraft.java):
+    - ChangeQuantity
+    - SetRestockableInDays
+    - SetExpectedDelivery
+    - SetSupplyChannel
+    - inventory custom fields update actions:
+        - SetCustomType
+        - SetCustomField
+
+## How to use it?
```

Comment:
@heshamMassoud Do you mean e.g. `errorCallback` from sync options? I typed here only inventories' specific options. Since callbacks are placed in `BaseSyncOptions` maybe it is better to provide README in `commons` package that would describe issues common to all syncers, WDYT?